canonical/canonical.com
# Wd 13776 Dev Refresh /data/mysql

## Done

- Refreshed MySQL
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- [Design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=705-1977&node-type=CANVAS&t=t1mHVIAEfPatvrtR-0)
- [Copydoc](https://docs.google.com/document/d/1KQmCfb4ITlYHOAEoHUwxHExf73qkMSWsLHnFPwFWFag/edit)
- [Demo](https://canonical-com-1364.demos.haus/data/mysql)
## Issue / Card

Fixes # [WD-13776](https://warthogs.atlassian.net/browse/WD-13776)

## Screenshots

[if relevant, include a screenshot]


[WD-13776]: https://warthogs.atlassian.net/browse/WD-13776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
